### DISCUSSION TITLE
How to implement the "box" effect?

### DISCUSSION THREAD

**Discussion by suerta-git** [nolimits4web/atropos#25] [Q&A]
Hi team, this is a really cool library! 

Just wondering how can I implement/simulate the "box" effect as your game box demo shows. 
Especially for the 4 side surfaces of the box I have no idea about the implementation. And I also didn't find the relevant apis I can use.

It would be nice to have a code example (I am using react). Thanks!

**Comment by valerebron** [comment]
I try to reverse ingenering the demo code.
It use a list of class like .game-box-t, .game-box-b, .game-box-l, .game-box-r.
But for now I cant reproduce the effect ... still working on.

**Reply by solrmax** [reply]
You can find the website source code here : https://github.com/nolimits4web/atropos-website
